packages/tests/tests/light-push/peer_management.spec.ts

```diff
@@ -78,18 +79,21 @@ describe("Waku Light Push: Peer Management: E2E", function () {
     expect(response2.failures).to.have.length(1);
     expect(response2.failures?.[0].peerId).to.equal(peerToDisconnect);
 
-    // send another lightpush request -- renewal should have triggerred and new peer should be used instead of the disconnected one
+    // send another lightpush request
+    // reattempts to send should be triggerred
+    // then renewal should happen
+    // so one failure should exist
     const response3 = await waku.lightPush.send(encoder, {
       payload: utf8ToBytes("Hello_World")
     });
 
+    await delay(500);
```

Comment:
do we need this delay because of internal operation that we cannot wait? if so, maybe adding a comment would be good